backend/src/schema/resolvers/users.js

```diff
@@ -177,6 +177,7 @@ export default {
         'termsAndConditionsAgreedVersion',
         'termsAndConditionsAgreedAt',
         'allowEmbedIframes',
+        'showShoutsPublicly',
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-14T08:17:53Z" title="Thursday, November 14th 2019, 9:17:53 am +01:00">Nov 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -177,6 +177,7 @@ export default {
         'termsAndConditionsAgreedVersion',
         'termsAndConditionsAgreedAt',
         'allowEmbedIframes',
+        'showShoutsPublicly',
```


Very important that we not forget to add this! Great job catching this issue before it bites us @Kachulio1